src/core/Geometry.js

```diff
@@ -1368,15 +1366,16 @@ Geometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 		return this;
 
-	},
+	}
 
-	dispose: function () {
+	dispose() {
 
 		this.dispatchEvent( { type: 'dispose' } );
 
 	}
 
-} );
+}
 
+Geometry.prototype.isGeometry = true;
```

Comment:
could you use the `Object.defineProperty()` pattern demonstrated in #20110 across the files modified in this PR? 🙏

Also support for class fields is coming soon.

Reply:
OK.